icons/harvester/harvester-plain-wordmark.svg

```diff
@@ -0,0 +1 @@
-1.625 1.627h-3.05c-.61 0-1.12-.306-1.424-.814l-5.895-10.268h-4.78l4.575 7.828h2.137c.915 0 1.627.712 1.627 1.627s-.712 1.627-1.627 1.627h-3.05c-.61 0-1.12-.306-1.425-.814L37.21 47.885c-.102 0-.102 0-.102-.102-.101-.101-.101-.2-.101-.303v-.814c0-.102.101-.205.101-.307l.102-.1 6.404-11.083a1.62 1.62 0 0 1 1.424-.81h3.05c.916 0 1.628.71 1.628 1.624 0 .915-.712 1.627-1.627 1.627h-2.137l-4.574 7.83h4.78l5.894-10.271a1.62 1.62 0 0 1 1.424-.81h3.05c.915 0 1.625.71 1.625 1.624 0 .915-.71 1.627-1.625 1.627h-2.134l-4.577 7.83h4.78l5.896-10.271a1.62 1.62 0 0 1 1.424-.81h3.05c.916 0 1.626.71 1.626 1.624 0 .915-.71 1.627-1.625 1.627h-2.135l-4.576 7.83h4.777l5.795-10.472.104-.102.203-.205h.1c0-.102.103-.203.204-.203.102 0 .203-.1.305-.1h.102c.101-.101.203-.101.304-.101zm3.051 3.353 1.83 3.254h3.455l1.832-3.254h-7.117zm-2.847 1.627-3.66 6.203h3.763l1.727-2.949-1.83-3.254zm12.709 0-1.83 3.254 1.728 2.95h3.658l-3.556-6.204zm-8.032 4.98-1.728 2.95 1.728 2.949h3.455l1.73-2.95-1.73-2.948h-3.455zM66.693 48.7l3.66 6.303 1.83-3.252-1.726-3.05h-3.764zm16.268.102-1.729 2.949 1.932 3.252 3.559-6.201H82.96zm-7.93 4.574L73.1 56.631h7.218l-1.832-3.256h-3.455zm4.778 37.21c-4.067 0-6.407 1.832-6.407 4.882 0 3.355 2.95 4.575 5.391 5.388 1.627.509 2.947.915 2.947 2.237 0 .915-.508 1.625-2.236 1.625-2.542 0-3.05-1.522-2.744-2.844l-3.66.303c-.509 3.558 2.236 5.388 6.2 5.388 4.372 0 6.509-2.133 6.509-5.082 0-3.355-2.951-4.372-5.391-5.185-1.627-.61-2.947-1.018-2.947-2.34 0-1.017.711-1.523 2.033-1.523 1.932 0 2.541 1.015 2.338 2.947l3.762-.305c.508-2.542-.915-5.388-5.795-5.49zM0 90.79v16.37h4.066v-6.915H9.05v6.914h4.066V90.79H9.05v6.203H4.066V90.79H0zm20.13 0-5.59 16.37h3.558l1.017-3.153h5.489l1.015 3.152h4.067L24.197 90.79h-4.066zm11.184 0v16.37h4.067v-6.509h.914c.915 0 1.525 2.138 2.033 3.256 1.118 2.644 2.238 3.354 4.17 3.354.508 0 1.119-.102 1.627-.203l.303-2.848c-.204.102-.406.102-.61.102-1.423 0-1.829-1.524-2.54-2.948-.204-.61-.611-1.118-1.12-1.525 1.932-.61 3.254-2.44 3.254-4.473 0-3.05-2.135-4.576-5.389-4.576h-6.709zm12.811 0 5.691 16.37h3.762l5.592-16.37h-3.66l-3.05 9.252c-.202 1.017-.407 1.933-.509 2.95h-.101c-.102-1.017-.203-1.933-.508-2.95l-3.049-9.252h-4.168zm16.674 0v16.572h11.08v-3.05h-7.014v-3.862h5.694V97.4h-5.694v-3.56h6.81v-3.05H60.8zm26.025 0v3.05H91.3v13.42h4.066V93.84h4.574v-3.05H86.824zm14.844 0v16.572h11.082v-3.05h-7.016v-3.862h5.694V97.4h-5.694v-3.56h6.81v-3.05h-10.876zm13.217 0v16.37h4.066v-6.509h.916c.915 0 1.525 2.138 2.033 3.256 1.119 2.644 2.237 3.354 4.168 3.354.509 0 1.117-.102 1.625-.203l.307-2.848c-.203.102-.406.102-.61.102-1.423 0-1.829-1.524-2.54-2.948-.204-.61-.613-1.118-1.121-1.525 1.931-.61 3.253-2.44 3.253-4.473 0-3.05-2.135-4.576-5.388-4.576h-6.71zM35.38 93.74h1.93c1.118 0 2.033.506 2.033 1.929 0 1.728-1.423 2.137-2.541 2.137H35.38v-4.067zm83.57 0h1.932c1.118 0 2.033.506 2.033 1.929 0 1.728-1.423 2.137-2.541 2.137h-1.424v-4.067zm-96.99 1.015h.102c.101 1.017.204 1.93.51 2.846l1.118 3.457h-3.558L21.35 97.5c.305-.915.507-1.83.609-2.744z"/></svg>
```

Comment:
The color should probably be added here, so that the SVG version has the correct color.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#00A580"d="M38.125 20.436c-3.558 0-6.506 2.845-6.506 6.404v40.566c0 3.559 2.948 6.508 6.506 6.508h51.852c3.558 0 6.505-2.95 6.505-6.508V26.941c0-3.558-2.947-6.505-6.505-6.505H38.125zM70.15 34.264h13.014c.102.101.203.101.305.101h.101c.102 0 .203.1.305.1l.203.203h.104l.203.205.101.102 6.405 11.082c.101.101.101.101.101.203.102.101.102.203.102.406v.711c0 .102-.102.203-.102.305 0 .101 0 .203-.203.203l-6.404 11.082-.102.101-.205.205h-.101c0 .102-.1.204-.202.204-.101 0-.103.1-.205.1 0 .1-.101.103-.101.103-.102.101-.305.101-.406.101H70.251c-.102 0-.305-.101-.406-.101 0-.102-.102-.104-.102-.104-.102 0-.101-.1-.203-.1l-.205-.203h-.102l-.2-.205-.102-.101-5.899-10.268h-4.777l4.576 7.828h2.135c.915 0 1.625.712 1.625 1.627s-.71 1.627-1.625 1.627h-3.051c-.61 0-1.119-.306-1.424-.814l-5.896-10.268h-4.78l4.577 7.828h2.134c.915 0 1.625.712 1.625 1.627s-.71 1.627-1.625 1.627h-3.05c-.61 0-1.12-.306-1.424-.814l-5.895-10.268h-4.78l4.575 7.828h2.137c.915 0 1.627.712 1.627 1.627s-.712 1.627-1.627 1.627h-3.05c-.61 0-1.12-.306-1.425-.814L37.21 47.885c-.102 0-.102 0-.102-.102-.101-.101-.101-.2-.101-.303v-.814c0-.102.101-.205.101-.307l.102-.1 6.404-11.083a1.62 1.62 0 0 1 1.424-.81h3.05c.916 0 1.628.71 1.628 1.624 0 .915-.712 1.627-1.627 1.627h-2.137l-4.574 7.83h4.78l5.894-10.271a1.62 1.62 0 0 1 1.424-.81h3.05c.915 0 1.625.71 1.625 1.624 0 .915-.71 1.627-1.625 1.627h-2.134l-4.577 7.83h4.78l5.896-10.271a1.62 1.62 0 0 1 1.424-.81h3.05c.916 0 1.626.71 1.626 1.624 0 .915-.71 1.627-1.625 1.627h-2.135l-4.576 7.83h4.777l5.795-10.472.104-.102.203-.205h.1c0-.102.103-.203.204-.203.102 0 .203-.1.305-.1h.102c.101-.101.203-.101.304-.101zm3.051 3.353 1.83 3.254h3.455l1.832-3.254h-7.117zm-2.847 1.627-3.66 6.203h3.763l1.727-2.949-1.83-3.254zm12.709 0-1.83 3.254 1.728 2.95h3.658l-3.556-6.204zm-8.032 4.98-1.728 2.95 1.728 2.949h3.455l1.73-2.95-1.73-2.948h-3.455zM66.693 48.7l3.66 6.303 1.83-3.252-1.726-3.05h-3.764zm16.268.102-1.729 2.949 1.932 3.252 3.559-6.201H82.96zm-7.93 4.574L73.1 56.631h7.218l-1.832-3.256h-3.455zm4.778 37.21c-4.067 0-6.407 1.832-6.407 4.882 0 3.355 2.95 4.575 5.391 5.388 1.627.509 2.947.915 2.947 2.237 0 .915-.508 1.625-2.236 1.625-2.542 0-3.05-1.522-2.744-2.844l-3.66.303c-.509 3.558 2.236 5.388 6.2 5.388 4.372 0 6.509-2.133 6.509-5.082 0-3.355-2.951-4.372-5.391-5.185-1.627-.61-2.947-1.018-2.947-2.34 0-1.017.711-1.523 2.033-1.523 1.932 0 2.541 1.015 2.338 2.947l3.762-.305c.508-2.542-.915-5.388-5.795-5.49zM0 90.79v16.37h4.066v-6.915H9.05v6.914h4.066V90.79H9.05v6.203H4.066V90.79H0zm20.13 0-5.59 16.37h3.558l1.017-3.153h5.489l1.015 3.152h4.067L24.197 90.79h-4.066zm11.184 0v16.37h4.067v-6.509h.914c.915 0 1.525 2.138 2.033 3.256 1.118 2.644 2.238 3.354 4.17 3.354.508 0 1.119-.102 1.627-.203l.303-2.848c-.204.102-.406.102-.61.102-1.423 0-1.829-1.524-2.54-2.948-.204-.61-.611-1.118-1.12-1.525 1.932-.61 3.254-2.44 3.254-4.473 0-3.05-2.135-4.576-5.389-4.576h-6.709zm12.811 0 5.691 16.37h3.762l5.592-16.37h-3.66l-3.05 9.252c-.202 1.017-.407 1.933-.509 2.95h-.101c-.102-1.017-.203-1.933-.508-2.95l-3.049-9.252h-4.168zm16.674 0v16.572h11.08v-3.05h-7.014v-3.862h5.694V97.4h-5.694v-3.56h6.81v-3.05H60.8zm26.025 0v3.05H91.3v13.42h4.066V93.84h4.574v-3.05H86.824zm14.844 0v16.572h11.082v-3.05h-7.016v-3.862h5.694V97.4h-5.694v-3.56h6.81v-3.05h-10.876zm13.217 0v16.37h4.066v-6.509h.916c.915 0 1.525 2.138 2.033 3.256 1.119 2.644 2.237 3.354 4.168 3.354.509 0 1.117-.102 1.625-.203l.307-2.848c-.203.102-.406.102-.61.102-1.423 0-1.829-1.524-2.54-2.948-.204-.61-.613-1.118-1.121-1.525 1.931-.61 3.253-2.44 3.253-4.473 0-3.05-2.135-4.576-5.388-4.576h-6.71zM35.38 93.74h1.93c1.118 0 2.033.506 2.033 1.929 0 1.728-1.423 2.137-2.541 2.137H35.38v-4.067zm83.57 0h1.932c1.118 0 2.033.506 2.033 1.929 0 1.728-1.423 2.137-2.541 2.137h-1.424v-4.067zm-96.99 1.015h.102c.101 1.017.204 1.93.51 2.846l1.118 3.457h-3.558L21.35 97.5c.305-.915.507-1.83.609-2.744z"/></svg>
```